astropy/cosmology/core.py

```diff
@@ -263,6 +263,13 @@ def __init__(self, H0, Om0, Ode0, Tcmb0=2.725, Neff=3.04,
             if self._massivenu:
                 nu_y = self._massivenu_mass / (kB_evK * self._Tnu0)
                 self._nu_y = nu_y.value
+                # This may seem strange, but it turns out to
+                # be much faster to use _nu_y as a python list than
+                # an ndarray for scalar evaluations; see
+                # _massivenu_relative_density_scalar.  So we provide
+                # both -- the ndarray for when z is an ndarray, and
+                # the list version for inside the integrals
+                self._nu_ylist = self._nu_y.tolist()
```

Comment:
Actually, not surprising at all now that I look at how this is used.  When we're talking just (typically) 3 neutrino species, looping over a simple Python list (in Python) is going to involve far less overhead than looping (again, in Python) over a Numpy array.